Gemfile

```diff
@@ -3,6 +3,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.2'
 
+# Tools or gems used to this current project
+gem 'cancancan', '~> 3.3'
+gem 'devise', '~> 4.8', '>= 4.8.1'
+gem 'font-awesome-rails'
+gem 'rails-controller-testing'
+gem 'rubocop', '>= 1.0', '< 2.0'
+
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails', branch: 'main'
 gem 'rails', '~> 6.1.4', '>= 6.1.4.3'
 # Use postgresql as the database for Active Record
```

Comment:
- [ ] I try to find where pg was used as the database for the current application, but could not see please if that is the case I would advise you to configure our current application use to pg. but ignore if that has been taken care of 🤓

Reply:
You can find it on config/database.yml